selftests/unit/test_utils_iso9660.py

```diff
@@ -43,8 +43,8 @@ def basic_workflow(self):
         self.iso.close()
         self.iso.close()    # check that double-close won't fail
 
-    @unittest.skipIf(not process.can_sudo(),
-                     "This test requires sudo or root")
+    @unittest.skipIf(not process.can_sudo("mount"),
+                     "This test requires mount to run under sudo or root")
```

Comment:
There's another place missing a similar change, it was causing the test to be skipped on my setup:

```diff
diff --git a/selftests/unit/test_utils_iso9660.py b/selftests/unit/test_utils_iso9660.py
index ec8e8ea..e4a0f4a 100644
--- a/selftests/unit/test_utils_iso9660.py
+++ b/selftests/unit/test_utils_iso9660.py
@@ -116,7 +116,7 @@ class IsoMount(BaseIso9660):
     Mount-based check
     """
 
-    @unittest.skipIf(not process.can_sudo(),
+    @unittest.skipIf(not process.can_sudo("mount"),
                      "This test requires sudo or root")
     def setUp(self):
         super(IsoMount, self).setUp()
```